src/loader.ts

```diff
@@ -42,8 +83,8 @@ export const LoadIconSet = UniqPromise(async (ctx: ExtensionContext, id: string)
         Log.info(`☁️ [${id}] Downloading from ${url}`)
         data = await $fetch(url)
         Log.info(`✅ [${id}] Downloaded`)
-        ctx.globalState.update(key, data)
-        LoadedIconSets[id] = data!
+        writeCache(ctx, id, data)
```

Comment:
```suggestion
        await writeCache(ctx, id, data)
```

Reply:
I don't think we should wait for writing the cache on both cases